File: Latihan_a/Ram.cpp
#include "Disk.cpp"

class Ram {

    private:
        // atribut
        string capacity;
        int price;

    public:
        // konstruktor
        Ram(){}

        Ram(string capacity, int price)
        {
            this->capacity = capacity;
            this->price = price;
        }

        // setter
        void setCapacity(string capacity)
        {
            this->capacity = capacity;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // getter
        string getCapacity()
        {
            return this->capacity;
        }

        int getPrice()
        {
            return this->price;
        }

        // destruktor
        ~Ram(){}
};

File: Latihan_b/Php/Ram.php
<?php

class Ram {

    // atribut
    private $capacity;
    private $price;

    
    // konstruktor

    function __construct($capacity = "", $price = 0)
    {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // setter
    function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    // getter
    function getCapacity()
    {
        return $this->capacity;
    }

    function getPrice()
    {
        return $this->price;
    }
}

?>